pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -9,12 +9,19 @@ const (
 	SecurityContextConstraintPrivileged = "privileged"
 	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
+	SecurityContextConstraintHostMount = "hostmount"
```

Comment:
The recycler creates a pod that runs "rm -rf" on the volume.  The pod has to mount the PV.  

If we want to support the recycler making HostPath recycling pods, it needs this permission.